synopsys-task/src/main.ts

```diff
@@ -72,19 +83,33 @@ export async function run() {
   console.log("Synopsys Task workflow execution completed");
 }
 
-export function logBridgeExitCodes(message: string): string {
-  const exitCode = message.trim().slice(-1);
+export function logBridgeExitCodes(message: string, exitCode: string): string {
```

Comment:
maybe we can rename it to logExitCodes instead of logBridgeExitCodes as we are logging both ADO and bridge exit codes.